kin-sdk-core/src/main/java/kin/sdk/core/KinClient.java

```diff
@@ -25,7 +22,7 @@
      * smart-contract problems.
      */
     public KinClient(Context context, ServiceProvider provider) throws EthereumClientException {
```

Comment:
We need to change `EthereumClientException` to be more generic like `ClientException`

Reply:
indeed, already done at a different [branch|https://github.com/kinfoundation/kin-sdk-core-stellar-android/blob/error_handling_enhancements/kin-sdk-core/src/main/java/kin/sdk/core/exception/ClientException.java]

Reply:
👍